src/Commands/InitCommand.php

```diff
@@ -6,11 +6,12 @@
 use Illuminate\Contracts\Console\Isolatable;
 use Illuminate\Support\Arr;
 use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Artisan;
 use Illuminate\Support\Facades\Validator;
 use Illuminate\Support\Str;
+use Illuminate\View\View;
```

Comment:
Contract have description method render
```suggestion
use Illuminate\Contracts\View\View;
```